harvard-lts/etd_alma_service
# ETD-334: Update Mongo with indash parameter

**ETD-334: Update Mongo with indash parameter**
* * *

**JIRA Ticket**: https://at-harvard.atlassian.net/browse/ETD-334

# What does this Pull Request do?
Update Mongo with indash parameter

# How should this be tested?

1. download branch
2. get alma_proquest_id_rsa & known_hosts from vault and put in .ssh/
3. copy env-example to .env and set to dev settings, ensure mongo values are set
4. start container docker-compose -f docker-compose-local.yml up -d --build --force-recreate
5. pytest. All tests should pass
6. launch compassdb or other mongo client. go to etd dev db, & look at the integration_test collection. you should see a new record with the "indash" field.
7. shutdown container docker-compose -f docker-compose-local.yml down 


# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? y
- integration tests? y

# Interested parties
Tag (@ mention) interested parties: @awoods
